hnn_core/optimization/optimize_evoked.py

```diff
@@ -356,7 +358,7 @@ def _optrun(drive_params_updated, drive_params_static, net, tstop, dt,
 
     opt_params['optiter'] += 1
 
-    return avg_rmse
+    return avg_rmse, avg_rmse_unweighted  # nlopt expects error
```

Comment:
Something went wrong in your last rebase. This is undoing a bug patch I made in my last PR.

Reply:
Not sure what went wrong but looking into it now. All other tests are passing so I think we'll be able to merge after this.